### ISSUE TITLE
EN Locale Missing Completely For /cmi banlist

### ISSUE THREAD

**Issue by xxirs** [Zrips/CMI#5495]
ban someone with /cmi ban
then do /cmi banlist
![image](https://user-images.githubusercontent.com/47906826/116132799-954a2600-a693-11eb-904f-53625f959d2a.png)

---

![Untitled](https://user-images.githubusercontent.com/47906826/116132984-cfb3c300-a693-11eb-9eb1-86f84d7692be.png)

**Comment by mrfloris** [comment]
Yep, same here.

**Comment by CloudeLecaw** [comment]
jep. looking in the langfile also shows this lines is missing if it should exist.

```
  banlist:
    help:
      info: '&ezeigt die Serverbanliste'
      args: ''

  ipbanlist:
    help:
      info: '&eBan ip list'
      args: ''
    info:
      banList: '&7[ip] &e- &6[reason]  '
      bannedBy: '&7Banned by: &f[source]'
      ends: '&7Ends: &f[time]'
      related: |-
        &7Related accounts:
        &f[list]
```

**Comment by CloudeLecaw** [comment]
may it should be 
command.ipbanlist.info.banList instead of command.ipbanlist.info.list

paste a coppy of the string banList and rename it to list.

```
  ipbanlist:
    help:
      info: '&eBan ip list'
      args: ''
    info:
      banList: '&7[ip] &e- &6[reason]  '
      list: '&7[ip] &e- &6[reason]  '
      bannedBy: '&7Banned by: &f[source]'
      ends: '&7Ends: &f[time]'
      related: |-
        &7Related accounts:
        &f[list]
```

**Comment by Zrips** [comment]
Fixed with 8.8.3.7

**Comment by mrfloris** [comment]
Ah nice! Thank you for looking into this.
